### PR TITLE
[hi] Localize /en/docs/reference/glossary/operator-pattern.md

### LINKED ISSUE #37855: [hi] Localize /en/docs/reference/glossary/operator-pattern.md
### Prerequisites:

- If this is your first contribution, visit [Contribute to K8s docs](https://kubernetes.io/docs/contribute/) for information on how to get started.
- Review a few https://github.com/kubernetes/website/issues/35600#issuecomment-1214215859 to see which 
  content you have to change and which you leave unchanged.

### Procedure:

- Make sure your base, target branch is `main` and create a working branch called `hi-operator-pattern.md`.
- Create a file called content/hi/docs/reference/glossary/operator-pattern.md
- Copy the content of 
  https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/operator-pattern.md 
  to the file you created
- Localize the content.
- Push your change and create a pull request.

Completing this feature issue fixes part of https://github.com/kubernetes/website/issues/35600

> **Note**:- Don't forget to join [`slack`](https://communityinviter.com/apps/kubernetes/community) `kubernetes-docs-hi` channel if facing any problem!

> **Note**:- As we are creating this issues for new contributors to help them. Please follow this guideline that `one time get assigned one issue only`. After completion that you can take another one. Follow this approach! **This issue is for first-time-contributor only**
 

/language hi
/triage accepted

### PR DESCRIPTION
**Pull request by Omkar0114**
[hi] Localize /en/docs/reference/glossary/operator-pattern.md
fixes #37855 
Signed-off-by: Omkar Kulkarni [opkulkarni0104@gmail.com](mailto:opkulkarni0104@gmail.com)

### CODE REVIEW

File: content/hi/docs/reference/glossary/operator-pattern.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+- architecture
+---
+[ऑपरेटर पैटर्न](/docs/concepts/extend-kubernetes/operator/) एक सिस्टम डिज़ाइन है जो {{< glossary_tooltip text="नियंत्रक" term_id="controller" >}}
+को एक या अधिक कस्टम संसाधनों से जोड़ता है।
+
+<!--more-->
+आप कुबेरनेट्स के हिस्से के रूप में आने वाले अंतर्निहित नियंत्रकों से परे, अपने क्लस्टर में नियंत्रकों को जोड़कर कुबेरनेट्स का विस्तार कर सकते हैं।
+
+यदि कोई चल रहा एप्लिकेशन नियंत्रक के रूप में कार्य करता है और उसके पास नियंत्रण विमान में परिभाषित कस्टम संसाधन के विरुद्ध कार्य करने के लिए एपीआई पहुंच है, तो यह ऑपरेटर पैटर्न का एक उदाहरण है।
```
Comment (kundan2707) [inline content/hi/docs/reference/glossary/operator-pattern.md:19]: @Omkar0114  translation of this line is not correct completely. Rewrite this line

Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md:19]: Which line are you saying exactly? @kundan2707

**Review by Arindam200 — APPROVED** [review]

File: content/hi/docs/reference/glossary/operator-pattern.md (hunk 1 above)
Comment (kundan2707) [reply content/hi/docs/reference/glossary/operator-pattern.md:19]: @Omkar0114 यदि कोई चल रहा एप्लिकेशन नियंत्रक के रूप में कार्य करता है और उसके पास नियंत्रण विमान में परिभाषित कस्टम संसाधन के विरुद्ध कार्य करने के लिए एपीआई पहुंच है, तो यह ऑपरेटर पैटर्न का एक उदाहरण है।

Comment (Babapool) [inline content/hi/docs/reference/glossary/operator-pattern.md:19]: Incorporating @kundan2707 's suggestion on this line as well:
```suggestion
यदि कोई चल रहा एप्लिकेशन नियंत्रक के रूप में कार्य करता है और उसके पास कंट्रोल प्लेन में परिभाषित कस्टम संसाधन पर कार्य करने के लिए API पहुंच की है, तो यह ऑपरेटर पैटर्न का एक उदाहरण है।
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from my end, hope you find them useful. Thanks

File: content/hi/docs/reference/glossary/operator-pattern.md (hunk 1 above)
Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md]: Done with this line @kundan2707 ✅

File: content/hi/docs/reference/glossary/operator-pattern.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+date: 2019-05-21
+full_link: /docs/concepts/extend-kubernetes/operator/
+short_description: >
+  कस्टम संसाधन का प्रबंधन करने के लिए उपयोग किया जाने वाला एक विशेष नियंत्रक
+
+aka:
+tags:
+- architecture
+---
+[ऑपरेटर पैटर्न](/docs/concepts/extend-kubernetes/operator/) एक सिस्टम डिज़ाइन है जो {{< glossary_tooltip text="नियंत्रक" term_id="controller" >}}
```
[the remark this answers is not in the document]
Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md]: Updated @Babapool

File: content/hi/docs/reference/glossary/operator-pattern.md (hunk 1 above)
Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md]: Updated with your suggestions @Babapool ✅Thanks for suggesting!

**Comment by bishal7679** [comment]
part of #35600

**Comment by Omkar0114** [comment]
Hi @anubha-v-ardhan , @verma-kunal 
 Can you please review the PR once, I updated it with the previous suggestions!
Thank you :)

**Comment by bishal7679** [comment]
PHAL!
cc @divya-mohan0209

**Comment by Omkar0114** [comment]
Hey @divya-mohan0209 Have a look once!

File: content/hi/docs/reference/glossary/operator-pattern.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+aka:
+tags:
+- architecture
+---
+[ऑपरेटर पैटर्न](/docs/concepts/extend-kubernetes/operator/) एक सिस्टम रचना है जो {{< glossary_tooltip text="नियंत्रक" term_id="controller" >}}
+को एक या अधिक कस्टम संसाधनों से जोड़ता है।
+
+<!--more-->
+आप कुबेरनेट्स के हिस्से के रूप में आने वाले अंतर्निहित नियंत्रकों से परे, अपने क्लस्टर में नियंत्रकों को जोड़कर कुबेरनेट्स का विस्तार कर सकते हैं।
```
[the remark this answers is not in the document]
Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md:17]: Yes, what do @verma-kunal @anubha-v-ardhan think?

[the remark this answers is not in the document]
Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md]: done with changes @Babapool

**Comment by Omkar0114** [comment]
Hey, @divya-mohan0209 Have a look once!

File: content/hi/docs/reference/glossary/operator-pattern.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+title: ऑपरेटर पैटर्न(Operator pattern)
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/operator-pattern.md:2]:
```suggestion
title: ऑपरेटर पैटर्न
```

Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md]: Done :)

**Review by Babapool — COMMENTED** [review]
Thanks!
/lgtm

File: content/hi/docs/reference/glossary/operator-pattern.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+aka:
+tags:
+- architecture
+---
+[ऑपरेटर पैटर्न](/docs/concepts/extend-kubernetes/operator/) एक सिस्टम रचना है जो {{< glossary_tooltip text="नियंत्रक" term_id="controller" >}}
+को एक या अधिक कस्टम संसाधनों से जोड़ता है।
+
+<!--more-->
+आप अंतर्निहित नियंत्रकों का उपयोग करने के अलावा जो स्वयं कुबेरनेट्स का हिस्सा हैं, अपने क्लस्टर में नियंत्रकों को जोड़कर कुबेरनेट्स की कार्यक्षमता का विस्तार कर सकते हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/operator-pattern.md:17]: Rephrasing it for better readability.
```suggestion
आप अंतर्निहित नियंत्रक, जो स्वयं कुबेरनेट्स का हिस्सा हैं, का उपयोग करने से परे, अपने क्लस्टर में नियंत्रकों को जोड़कर कुबेरनेट्स की कार्यक्षमता का विस्तार कर सकते हैं।
```

File: content/hi/docs/reference/glossary/operator-pattern.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+यदि कोई चल रहा एप्लिकेशन नियंत्रक के रूप में कार्य करता है और उसके पास कंट्रोल प्लेन में परिभाषित कस्टम संसाधन पर कार्य करने के लिए API पहुंच की है, तो यह ऑपरेटर पैटर्न का एक उदाहरण है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/operator-pattern.md:19]:
```suggestion
यदि कोई चालू एप्लिकेशन नियंत्रक के रूप में कार्य करता है और उसके पास कंट्रोल प्लेन में परिभाषित कस्टम संसाधन पर कार्य करने के लिए API अभिगम है, तो यह ऑपरेटर पैटर्न का एक उदाहरण है।
```

File: content/hi/docs/reference/glossary/operator-pattern.md (hunk 5 above)
Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md]: Done with suggestion !

File: content/hi/docs/reference/glossary/operator-pattern.md (hunk 6 above)
Comment (Omkar0114) [reply content/hi/docs/reference/glossary/operator-pattern.md]: Updated :)

**Comment by Omkar0114** [comment]
Updated with suggestion, check out once @divya-mohan0209

**Review by divya-mohan0209 — COMMENTED** [review]
Using LGTM from Vitthal above since content was only rearranged + tense was changed for improved readability.
/lgtm
/approve

**Comment by Omkar0114** [comment]
Thanks for merging @divya-mohan0209 @Babapool  🥳

**Pull request merged**